### PR TITLE
Adds an admin task for Boba

### PR DESCRIPTION
**Pull request by sarthaktexas**
## Summary of the problem

Max wants to add this task.

## Describe your changes

Adds a task.

image

### CODE REVIEW

File: app/helpers/static_pages_helper.rb [hunk 1]
```diff
@@ -144,6 +144,12 @@ def airtable_info
         table: "Google%20Workspace%20Waitlist",
         query: { filterByFormula: "Status='Pending'" },
         destination: "https://airtable.com/appEzv7w2IBMoxxHe/tbl9CkfZHKZYrXf1T/viwgfJvrrD9Jn9VLj"
+      },
+      boba: {
+        id: "app05mIKwNPO2l1vT",
+        table: "Event Codes",
```
Comment (sarthaktexas) [inline app/helpers/static_pages_helper.rb]:
```suggestion
        table: "Event%20Codes",
```

**Review by sampoder — COMMENTED** [review]
Screenshot 2024-05-22 at 2 57 25 PM

Screenshot 2024-05-22 at 2 57 38 PM

It seems to be erroring :( I'm not sure why yet

**Comment by garyhtou** [comment]
If it's erroring, you may need to configure airbridge to allow that base.

also, is operations processing Boba drops?

**Comment by garyhtou** [comment]
Is this something we can close out in the next few days?

**Pull request merged**